babel.config.json

```
@@ -0,0 +1,12 @@
+{
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "targets": {
+          "node": "14"
```

```suggestion
          "node": "18"
```

Should we bump this to the min. supported Node version?